backend/core/tests/test_task.py

```
@@ -0,0 +1,288 @@
+from datetime import date, datetime, timedelta
```

Ruff (0.8.2)

`datetime.datetime` imported but unused

Remove unused import

(F401)